packages/@biomejs/biome/configuration_schema.json

```diff
@@ -83,98 +83,98 @@
 				"noAccessKey": {
 					"description": "Enforce that the accessKey attribute is not used on any HTML element.",
 					"anyOf": [
-						{ "$ref": "#/definitions/RuleConfiguration" },
+						{ "$ref": "#/definitions/RuleConfiguration_for_Null" },
```

Comment:
I think it's a bit unfortunate these namings are leaking into the schema now. Could we keep `RuleConfiguration_for_Null` with the old `RuleConfiguration` and rename types like `RuleWithOptions_for_ConsistentArrayTypeOptions` to something like `ConsistentArrayTypeRuleConfiguration`?

Reply:
I am not sure if it is easily modifiable. schemars is hard to customize.

Reply:
Oh yeah, I ran into that with my `Partial` refactor as well. You may want to look at the `rename_partial_references_in_schema()` function, since I opted to just rename references in the schema after it was generated :)

Reply:
```
RuleConfiguration_for_Null  ->  RuleConfiguration
RuleWithOptions_for_Null  ->  RuleWithNoOptions
RuleConfiguration_for_XOptions  ->  XConfiguration
RuleWithOptions_for_XOptions  ->  RuleWithXOptions
RuleConfiguration_for_X  ->  XConfiguration
RuleWithOptions_for_X  ->  RuleWithX
```

Reply:
Not too opinionated on the exact phrases, but I can pick two :)

```
RuleWithOptions_for_Null  ->  RuleWithNoOptions
RuleWithOptions_for_XOptions  ->  RuleWithXOptions
```

Reply:
Oh wait, they're all the variations that currently exist? Guess that's fine then :+1:

Reply:
The two last ones doesn't exist. However, it is future-proof if a dev doesn't suffix a rule options with `Options`